lib/galaxy/tools/deps/resolvers/__init__.py

```diff
@@ -100,5 +107,12 @@ def __init__(self, version=None, name=None):
         self.version = version
         self.name = name
 
+    @property
+    def resolver_msg(self):
+        """
+        Return a message describing this dependency
+        """
+        return "Dependency %s not found." % self.name
```

Comment:
Should we log the version too here? I can imagine the case where the same dependency is required multiple times in different versions for different resolvers.